internal/ctrlutils/ingress-status.go

```diff
@@ -440,11 +469,15 @@ func UpdateKnativeIngress(ctx context.Context, logger logr.Logger, svc file.FSer
 				break
 			}
 			if errors.IsNotFound(err) {
+				log.Debugf("failed to update Knative Ingress status because the object (%s/%s) is gone, status update stopped.", namespace, name)
 				return nil
 			}
-
-			log.Errorf("failed to update ingress status: %v. retrying...", err)
-			time.Sleep(time.Second)
+			if errors.IsConflict(err) {
+				log.Debugf("failed to update Knative Ingress status because the object (%s/%s) changed. retrying...", namespace, name)
```

Comment:
```suggestion
				log.Debugf("failed to update Knative Ingress (%s/%s) status, err  %v", namespace, name, err)
```

Reply:
:+1: changed in 2812019f36e6aed24e3021ec977f0475ab68e90d